go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	golang.org/x/exp v0.0.0-20230510235704-dd950f8aeaea
 	golang.org/x/text v0.13.0
 	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	software.sslmate.com/src/go-pkcs12 v0.4.0
```

Comment:
For whatever reason when I added this module in https://github.com/OctopusDeployLabs/terraform-provider-octopusdeploy/pull/636 it added it as an indirect module so makes sense that it changed to a direct one here